__tests__/config.test.ts

```diff
@@ -33,15 +33,15 @@ const FIXTURES_DIR = join(__dirname, 'fixtures');
 describe('Config', () => {
   it('read config based on environment', async () => {
     const configPath = join(FIXTURES_DIR, 'synthetics.config.ts');
-    expect(readConfig('development', configPath)).toMatchObject({
+    expect(await readConfig('development', configPath)).toMatchObject({
```

Comment:
We need a new test where fetch these configs remotely and merge it inside params and also catching these errors correctly.